src/common/m_variables_conversion.fpp

```diff
@@ -1158,13 +1133,13 @@ contains
                         end do
 
                         call get_mixture_molecular_weight(Ys, mix_mol_weight)
-                        T = q_prim_vf(E_idx)%sf(j, k, l)*mix_mol_weight/(gas_constant*rho)
+                        T = q_prim_vf(tempxb)%sf(j, k, l)
```

Comment:
tempxb -> tempx_beg (following other nomenclature in the code)?

Reply:
The nomenclature in the code is actually:
- `E_idx` for Energy/pressure. This is because there is always only one such equation. Fixed.
- `<var>xb` and `<var>xe` for all other variables.

So I'm changing the use for temperature to `T_idx` to follow the current nomenclature, based on a sample size of 1.